migtests/tests/pg/pgtbrus/expected_files/expected_failed.sql

```diff
@@ -41,6 +29,18 @@ OPTIONS (
     table_name 't'
 );
 
+/*
```

Comment:
why did this move down, suddenly?

Reply:
Okay, but the filename still says table.sql below?

Reply:
yeah, I didn't change it as we ignore the file name while validating these files I just moved it as is.

Reply:
I think it's just there, and the file path can vary that's why it's not getting validated. cc: @shubham-yb

Reply:
Yes